docs/pages/blog/meet-joy.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Meet Joy UI－a new starting point to your design system
+description: Wip description.
+date: 2022-06-01T00:00:00.000Z
+authors: ['danilo-leal', 'siriwatknp']
+tags: ['News']
+card: true
+---
+
+:::warning
+**work in progress**
+:::
+
+If you're a close follower of everything MUI, you might be aware that we've been working on a new design system product. If not, we're writing this piece to give you a first look at Joy－MUI's new starting point for your design system!
+
+## How this is relevant to you?
+
+You might intuitively associate MUI with Material UI, which despite the indeed potentially confusing names, are separate entities. Material UI is MUI's React implementation of Google's Material Design and over the years has established itself as a go-to library for quickly breathing life into products, mostly due to the number of components you get out of the box. However, the components do come with that default 2014 Google look and feel that a good portion of you grew to dislike. And, as we've reconfirmed with our latest developer survey, the design aspect of UI libraries is one of the most important ones whenever evaluating which to go for.
```

Comment:
```suggestion
You might intuitively associate MUI with Material UI, which despite the indeed potentially confusing names, are separate things. Material UI is MUI's React implementation of Google's Material Design. Over the years Material UI has established itself as the go-to library for quickly breathing life into products, mostly thanks to its design, customizability, and documentation. However, the components do come with the default 2018 Google look and feel that is no longer as popular as it was. And, as we've reconfirmed with our latest developer survey, the design aspect of UI libraries is one of the most important ones whenever evaluating which to choose.
```

> mostly due to the number of components you get out of the box

According to the survey it's:

1. Design
2. Customizability
3. Documentation

https://mui.com/blog/2021-developer-survey-results/#what-are-your-most-important-criteria-for-choosing-a-ui-library